roles-and-responsibilities.md

```diff
@@ -0,0 +1,17 @@
+# Roles and Responsibilities
+
+## Chairperson
+
+The FinOps Foundation Technical Advisory Council chairperson responsibilities include:
+
+  * Helping build a technical vision for the Foundation updated quarterly on a 12 month rolling basis
+  * Being the member voice leading and setting the goals of the TAC
+  * Collaborating with the Governing Board for funding initiatives that require funding
+  * Helping to determine what new programs to launch
+  * Providing guidance and assistance to working groups, special interest groups and other initiatives that emerge from the TAC
```

Comment:
This  line about guidance feels relevant to the TAC liaison to a WG/SG not the TAC chair